static/js/icebox-web.js

```diff
@@ -324,7 +324,7 @@ function drinkSelection(barcode, callback) {
         }
       },
       danger: {
-        label: "Noooooooooo",
+        label: "Abbrechen",
```

Comment:
Diese UX-Änderung widerspricht den Richtlinien für die Gestaltung von Nutzeroberflächen (MS hatte da bereits in den 90ern sehr gute Guidelines für), die sagen, dass Bestätigungsfragen mit Ja/Nein zu beantworten sind. Entsprechend wäre das Ändern des Nein-Knopfes in einen Abbrechen-Knopf inkonsistent mit der sonst gewohnten Gestaltung solcher Oberflächen.